extension/goplugin/schemas_test.go

```diff
@@ -623,9 +624,9 @@ var _ = Describe("Schemas", func() {
 		)
 
 		BeforeEach(func() {
-			if os.Getenv("MYSQL_TEST") != "true" {
-				Skip("Locks are only valid in MySQL")
-			}
+
+			//TODO: tests must be fixed for MYSQL DB as that is implemented by default in CircleCI
```

Comment:
What has to be fixed to have those tests running?

Reply:
Not evaluated but it's not maintained long time. I think it's ok to be done after this PR because even now it's not running.

Reply:
Ok, I was asking more out of curiosity if this is an easy thing or not. I have no problem with fixing that in a future PR.